docs/source/development/index.rst

```diff
@@ -0,0 +1,56 @@
+👩‍💻 Development
+=================
+
+Quick start
+-----------
+
+To obtain a copy of the code and work on a new branch::
+
+    git clone https://github.com/NASA-PDS/data-upload-manager.git
+    git checkout -b "<issue number>_<issue name>"
+
+Create a virtual environment in ``venv`` using Python 3.9 or later::
+
+    python3 -m venv venv
+
+Install the package and its dependencies for development into the virtual environment::
+
+    pip install --editable '.[dev]'
+
+At this point, the command line tools are now available in ``venv/bin``
+as explained in `usage`_.
+
+Testing
+-------
+
+The code base includes unit tests. Once you've installed the service, you can
+run the unit tests with the following command:
+
+    venv/bin/tox py39
```

Comment:
I think `tox` will fail because it expects the `PATH` to contain the venv, and running it as `venv/bin/tox` won't set the `PATH`. But if you "activate" the venv, then it'll be fine—and the invocation is even simpler: `tox py39`.